### ISSUE TITLE
The plugin automatically re-generates new password every time

### ISSUE THREAD

**Issue by subokita** [Loupedeck-open-source/Loupedeck-ObsStudio-OpenPlugin#46]
After the latest update, it seems like the plugin automatically regenerates a new OBS websocket password every time the LoupeDeck config is rebooted, and this causes all my other apps that are connecting to the websocket to fail, requiring change of password.

Could you please revert back the behavior, so that it doesn't regenerate new password everytime?

**Comment by CaiqueLPMS** [comment]
Seems to be the same for me. Temporary solution which seems to work is disabling Authentication in OBS. Not nice, but well...

![Screenshot 2024-10-01 225552](https://github.com/user-attachments/assets/13548355-c172-4a8c-b22c-bd109ac37ee4)

**Comment by St0l3nID** [comment]
Yeah same here, took me forever to figure this out, right now i have to do a stupid routine of opening different programms in a certain order with a certain dealy and disable the authentication somehwere in the middle.

Either let us enter a password, or stop this automatic behaviour completly. This is not usable, loupedeck is one of 4 applications that need to communitcate with the obs websocket and having a new, random passsword on every OBS restart because of loupdeck is not useable.

**Comment by uabassguy** [comment]
Anyone else have a workaround for this? Having the same issue. We need a spot to add credentials to loupedeck.

**Comment by Eloujah** [comment]
This automatic behaviour is annoying when using other 3rd party or internally created plugins that actually use the websocket too.

**Comment by Xanacas** [comment]
#50 hoping my pull request gets through. I fixed this issue.
